.github/workflows/build.yml

```
@@ -3,57 +3,96 @@ name: Build
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - AGENTS.md
   pull_request:
     branches: [ main ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)